### PR TITLE
Add "not" button to Compendium Browser filter tags

### PR DESCRIPTION
**Pull request by In3luki**
No functional change just better UI:

Recording 2023-11-01 at 13 29 35

### CODE REVIEW

File: src/module/apps/compendium-browser/index.ts [hunk 1]
```diff
@@ -596,13 +608,15 @@ class CompendiumBrowser extends Application {
...
-                            const current = !!selected.not;
-                            selected.not = !current;
-                            this.render();
+                        const action = htmlClosest(target, "[data-action]")?.dataset?.action;
+                        if (action === "toggle-not") {
+                            const value = event.detail.data.value;
+                            const selected = data.selected.find((s) => s.value === value);
+                            if (selected) {
+                                const current = !!selected.not;
+                                selected.not = !current;
```
Comment (stwlam) [inline src/module/apps/compendium-browser/index.ts:617]: what in ternation

Comment (stwlam) [reply src/module/apps/compendium-browser/index.ts:617]: Is this `selected.not = !selected.not;`?

Comment (In3luki) [reply src/module/apps/compendium-browser/index.ts:617]: > Is this `selected.not = !selected.not;`?

Ehhh, probably? I saw that but assumed I did it that way for a reason. I'll see if anything breaks.

Comment (In3luki) [reply src/module/apps/compendium-browser/index.ts]: Don't know what happended there. It's gone now. You saw nothing.

**Pull request merged**